src/core.c

```diff
@@ -1348,6 +1357,9 @@ static int gpu_monitoring(Labels *data)
 			casprintf(&data->tab_graphics[VALUE][GPU1PCIE        + i * GPUFIELDS], false,
 				_("Current: PCIe Gen%1dx%d / Max: Gen%1dx%d"),
 				pcie_sta_gen, atoi(pcie_sta_width), pcie_max_gen, atoi(pcie_max_width));
+		if(!ret_vram_total)
+			casprintf(&data->tab_graphics[VALUE][GPU1REBAR       + i * GPUFIELDS], false,
+				_("%s"), (atol(vram_total) / divisor_vram * 0.9 < data->g_data->bar_size[i] / (1 << 20)) ? "Enabled" : "Disabled");
```

Comment:
```c
_("%s"), [...] ? "Enabled" : "Disabled");
```
It will not work as expected, because gettext will not extract these strings. Please use something like this instead:
```c
"%s", [...] ? _("Enabled") : _("Disabled"));
```

Reply:
Thanks for your review, I fixed it.
https://github.com/X0rg/CPU-X/pull/196/commits/043c2d1b08cddff60148b79e9c96281d836cc672